packages/router/src/__tests__/links.test.tsx

```diff
@@ -29,11 +30,13 @@ describe('<NavLink />', () => {
     const mockLocation = createDummyLocation('/dunder-mifflin')
 
     const { getByText } = render(
-      <LocationProvider location={mockLocation}>
-        <NavLink activeClassName="activeTest" to="/dunder-mifflin">
-          Dunder Mifflin
-        </NavLink>
-      </LocationProvider>
+      <RouterContextProvider>
```

Comment:
Not 100% sure if this would be a breaking change for people who might write tests to check that there links are active in app.  It's necessary because 

`useMatch` needs the RouterContext now to pass the trailingSlashes to the `matchPath` function